aiohttp/http_parser.py

```diff
@@ -225,6 +225,11 @@ def parse_headers(
         return (CIMultiDictProxy(headers), tuple(raw_headers))
 
 
+def _is_supported_upgrade(headers: CIMultiDictProxy[str]) -> bool:
+    """Check if the upgrade header is supported."""
+    return headers.get(hdrs.UPGRADE, "").lower() in ("tcp", "websocket")
```

Comment:
```suggestion
    return headers.get(hdrs.UPGRADE, "").lower() in {"tcp", "websocket"}
```

I'd probably use a set for best practices (I think when I tested performance in the past, it's only ~10% slower than checking the first item of a tuple and then significantly faster if needing to check anything more than 1 item).